src/CodeGeneration.Roslyn.Plugin.Sdk/build/BuildPluginPackage.targets

```diff
@@ -56,9 +56,12 @@
   <ItemGroup>
     <CodeGenerationRoslynPlugin Include="%24(MSBuildThisFileDirectory)../$(PluginPackagePath)/$(TargetFileName)" />  
   </ItemGroup>
-  <ItemGroup>
-    @(GeneratorBuildPropertyNames->'<GeneratorBuildPropertyNames Include="%(Identity)" />')
-  </ItemGroup>
+  <PropertyGroup>
+    <CodeGenerationRoslynRequestedProperties>
+      %24(CodeGenerationRoslynRequestedProperties);
+      $(CodeGenerationRoslynRequestedProperties)
```

Comment:
I think the property used in Plugin project should be named differently, e.g. `PluginPackageRequestedProperties` - as in, this should be the Property used in Plugin csproj to request these properties be added in the package `targets`.

In the target that creates the package targets (the line commented on), we add `PluginPackageRequestedProperties` to `CodeGenerationRoslynRequestedProperties`.

The distinction would be necessary if for example the Plugin project itself used the Tool and some Plugins to generate some internal code. In that case, current code would add RequestedProperties added by *plugins used by* our Plugin.

It's a lot like inception ;) If it's not clear, I can offer an example.

Reply:
Think I get it haha, please check 56a3641 and let me know if I did it right.

Reply:
LGTM - I'd prefer PluginRequestedProperties than PackageRequestedProperties if you want it shorter, because `Package*` prefix is mostly used by nuget pack targets - I'd avoid possibility of clash in future.

Reply:
Ah, ok! Renamed to PluginRequestedProperties in 982c210